### PR TITLE
Prison Camp Fence

### PR DESCRIPTION
**Pull request by MustaphaTR**
Moved Phrohdoh/OpenRA#77 to here.

 #1 had to be splited anyway. This can be deleted from there.

### CODE REVIEW

File: rules/civilian-structures.yaml [hunk 1]
```diff
@@ -49,6 +49,24 @@ gasand:
 	Crushable:
 		CrushClasses: fence
 
+
```
Comment (abcdefg30) [inline rules/civilian-structures.yaml]: Please remove this extra new line.

File: sequences/civilian-structures.yaml [hunk 2]
```diff
@@ -43,6 +43,23 @@ gasand:
 		Offset: 0,0
 		UseTilesetCode: false
 
+
```
Comment (abcdefg30) [inline sequences/civilian-structures.yaml]: Same here.

File: sequences/civilian-structures.yaml [hunk 3]
```diff
@@ -43,6 +43,23 @@ gasand:
...
+		ShadowStart: 32
+	damaged-idle: ctfncp
+		Start: 16
+		Length: 16
+		ShadowStart: 48
+	icon: wallicon
+		Offset: 0,0
+		UseTilesetCode: false
+
+
```
Comment (abcdefg30) [inline sequences/civilian-structures.yaml]: And here.

**Comment by MustaphaTR** [comment]
Updated.

**Comment by abcdefg30** [comment]
:+1:

**Comment by abcdefg30** [comment]
I further cleaned the entire wall yaml up (https://github.com/OpenRA/ra2/commit/ec6b78f632f4abd25cb65477bfbe95064a0f63b0) and also added the damaged/destroyed sounds (https://github.com/OpenRA/ra2/commit/796fb4069081c857bb5a838b3543b02f83d22c50).

**Pull request merged**